### PR TITLE
Document the bug in csv_export2 on year 2262

### LINKED ISSUE #473: Update csv_export2 documentation on dates limitation
https://github.com/equinor/semeio/blob/999b075c26d7494dea41c60b1fd6b9023d3448d4/semeio/workflows/csv_export2/csv_export2.py#L39. Add a comment: Do not support Dates beyond year 2262.

### PR DESCRIPTION
**Pull request by berland**
Resolves #473 

See also issue #474.

### CODE REVIEW

**Review by xjules — APPROVED** [review]
👍

File: semeio/workflows/csv_export2/csv_export2.py [hunk 1]
```diff
@@ -42,6 +42,9 @@
 requested in the Eclipse DATA file. A wildcard like ``W*`` can in certain cases
 (e.g. Eclipse simulations with 100+ wells) produce thousands of vectors, and can
 then be replaced by something more explicit like ``WOPT* WGPT* WWPT*``.
+
+Dates beyond year 2262 are not supported. Use ecl2csv in combination with
+merge_csv in such scenarios.
```
Comment (hnformentin) [inline semeio/workflows/csv_export2/csv_export2.py:47]: Would it be possible to add an example of this use?

**Comment by hnformentin** [comment]
After commenting I noticed it is actually already approved,  but tests are not passing. 
Could you please take a look in my comment and the tests?

File: semeio/workflows/csv_export2/csv_export2.py (hunk 1 above)
Comment (berland) [reply semeio/workflows/csv_export2/csv_export2.py:47]: I think it is covered by the examples provided by ecl2df and subscript (as forward models and workflows injected into the ert documentation in fmu-docs.

**Comment by berland** [comment]
I think a new release of ert is needed on pypi for the tests to pass.

**Pull request merged**